example/resources/ansible/setup.yml

```diff
@@ -0,0 +1,30 @@
+---
+- hosts: all
+  become: yes
+  tasks:
+    - name: restart consul
+      systemd:
+        name: consul
+        state: restarted
+    - name: restart nomad
+      systemd:
+        name: nomad
+        state: restarted
+    - name: wait for consul to come up
+      uri:
+        url: "http://localhost:8500/v1/catalog/nodes"
+        status_code: 200
+      register: result
+      until: result.status == 200
+      retries: 60
+      delay: 1
```

Comment:
I think 1 sec delay before first request, does not make sense :) IMHO 30 secs at least.

Reply:
I think this is delay between retires. 3 secs is better in that case. A total of 3 minutes.

Reply:
In my experience both services become available in a matter of seconds, so why do you mean 3 sec is better, Fredrik?

Reply:
[Thundering herd](https://en.wikipedia.org/wiki/Thundering_herd_problem#:~:text=In%20computer%20science%2C%20the%20thundering,able%20to%20handle%20the%20event.)